### ISSUE TITLE
Link-to-text contrast should be at least 3:1

### ISSUE THREAD

**Issue by Otterfan** [BCLibraries/bc-libraries-site#34]
WCAG 2.0 AAA says that links should contrast with surrounding text by at least a 3:1 ratio. Our current links (151515 text color vs 990000 background) only come in at 2:1.

A contrast-checker is available at http://webaim.org/resources/contrastchecker/.

See [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/G17).
